### ISSUE TITLE
'rawunicodeescape' codec can't decode bytes in position 80-81: truncated \uXXXX error in to_yaml and to_json

### ISSUE THREAD

**Issue by fera0013** [fchollet/keras#5180]
`model.to_json() ` and `model.to_yaml()` on the model

> ____________________________________________________________________________________________________
> Layer (type)                     Output Shape          Param #     Connected to                     
> ====================================================================================================
> lambda_1 (Lambda)                (None, 3, 160, 320)   0           lambda_input_1[0][0]             
> ____________________________________________________________________________________________________
> convolution2d_1 (Convolution2D)  (None, 1, 40, 16)     327696      lambda_1[0][0]                   
> ____________________________________________________________________________________________________
> elu_1 (ELU)                      (None, 1, 40, 16)     0           convolution2d_1[0][0]            
> ____________________________________________________________________________________________________
> convolution2d_2 (Convolution2D)  (None, 1, 20, 32)     12832       elu_1[0][0]                      
> ____________________________________________________________________________________________________
> elu_2 (ELU)                      (None, 1, 20, 32)     0           convolution2d_2[0][0]            
> ____________________________________________________________________________________________________
> convolution2d_3 (Convolution2D)  (None, 1, 10, 64)     51264       elu_2[0][0]                      
> ____________________________________________________________________________________________________
> flatten_1 (Flatten)              (None, 640)           0           convolution2d_3[0][0]            
> ____________________________________________________________________________________________________
> dropout_1 (Dropout)              (None, 640)           0           flatten_1[0][0]                  
> ____________________________________________________________________________________________________
> elu_3 (ELU)                      (None, 640)           0           dropout_1[0][0]                  
> ____________________________________________________________________________________________________
> dense_1 (Dense)                  (None, 512)           328192      elu_3[0][0]                      
> ____________________________________________________________________________________________________
> dropout_2 (Dropout)              (None, 512)           0           dense_1[0][0]                    
> ____________________________________________________________________________________________________
> elu_4 (ELU)                      (None, 512)           0           dropout_2[0][0]                  
> ____________________________________________________________________________________________________
> dense_2 (Dense)                  (None, 1)             513         elu_4[0][0]                      
> ====================================================================================================
> Total params: 720,497
> Trainable params: 720,497
> Non-trainable params: 0
____________________________________________________________________________________________________

throw the exception 

> 'rawunicodeescape' codec can't decode bytes in position 80-81: truncated \uXXXX

**Comment by fera0013** [comment]
Apparently a problem in my development environment and not in keras.

**Comment by djnugent** [comment]
What was the solution?

**Comment by usuyama** [comment]
@fera0013 I'm seeing issue on Windows Server 2016. Could you share your workaround?

**Comment by usuyama** [comment]
It appears the issue is forward slash on Windows 10/2016.

The answer from this stackoverflow worked.
 https://stackoverflow.com/questions/41847376/keras-model-to-json-error-rawunicodeescape-codec-cant-decode-bytes-in-posi
